agent/src/beerocks/monitor/monitor_thread.cpp

```diff
@@ -596,18 +596,18 @@ bool monitor_thread::create_ap_metrics_response(uint16_t mid,
             continue;
         }
 
-        if (!mon_stats.add_ap_metrics(cmdu_tx, bssid)) {
+        auto vap_node   = *mon_db.vap_get_by_id(vap_id);
```

Comment:
Shouldn't we check that this is not null?

Reply:
Fixed